### PR TITLE
Page destination

### PR DESCRIPTION
**Pull request by BENSFIA**
fin de css de la page destination

### CODE REVIEW

File: client/src/pages/DestinationPage.css [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  display: flex;
+  flex-direction: column;
+}
+
+/* .plane {
+  display: flex;
+  position: absolute;
+  left: 0;
+  height: 180px;
+} */
```
Comment (nimser) [inline client/src/pages/DestinationPage.css]: Please don't commit dead code (= commented out code)

File: client/src/pages/DestinationsPage.jsx [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
-	return (
-		<h1>Hello from Destination Page</h1>
-	)
-}
\ No newline at end of file
+  return (
+    <div className="page_destination">
+      {/* <div className="plane">
+        <img className="airplaine" src={airplane} alt="airplane" />
+      </div> */}
```
Comment (nimser) [inline client/src/pages/DestinationsPage.jsx]: Please don't commit dead (= commented out) code

**Review by nimser — CHANGES_REQUESTED** [review]
@BENSFIA please don't push dead code.

After you make these changes, please ask a member of your team to review (approve or request change) your work.

File: client/src/pages/DestinationPage.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 15px;
+  row-gap: 0px;
+  justify-content: center;
+  max-width: 965px;
+  margin: 32px auto;
+}
+*/ .title span {
```
Comment (nimser) [inline client/src/pages/DestinationPage.css]: CSS is broken here because `*/`

**Review by nimser — CHANGES_REQUESTED** [review]

**Review by Smail-al — APPROVED** [review]
Good job!

**Review by nimser — APPROVED** [review]
ok

**Pull request merged**